dashboard/settings/base.py

```diff
@@ -46,6 +46,7 @@
     'django.contrib.staticfiles',
     'django.contrib.humanize',
 
+    'debug_toolbar',
```

Comment:
we shouldn't keep this in `base.py`, as this will be deployed to staging and prod.